src/styles/globals.css

```
@@ -10,6 +10,81 @@ body {
+  [data-testid='settings-panel'] input[type='password'],
+  [data-testid='settings-panel'] input[type='number'],
+  [data-testid='settings-panel'] input[type='url'],
+  [data-testid='settings-panel'] input[type='email'],
+  [data-testid='settings-panel'] textarea,
+  [data-testid='settings-panel'] select {
+    @apply !border shadow-sm outline-none transition;
+    border-color: color-mix(
+      in srgb,
+      var(--color-primary) 24%,
+      var(--color-text-base)
+    ) !important;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)